content/updates/04-2020/2-associations-updates.md

```diff
@@ -0,0 +1,30 @@
+# Product associations update
+
+
+## View the number of associations per product
+::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="10th of April" link-to-doc="../articles/products-associations.html#view-the-products-associations"
```

Comment:
```suggestion
::: meta-data type="New" features="Product associations, Productivity, Data Governance" available="mid-April" link-to-doc="../articles/products-associations.html#view-the-products-associations"
```

Reply:
I agree not to be precise at a day level